core/src/test/kotlin/org/evomaster/core/output/service/TestSuiteWriterTest.kt

```diff
@@ -103,4 +98,49 @@ class TestSuiteWriterTest{
         assertTrue(methods.any { it.name == "tearDown" })
         assertTrue(methods.any { it.name == "initTest" })
     }
+
+
+    @Test
+    fun testPythonCreatesRequirementsFile(){
+
+        val injector = LifecycleInjector.builder()
+            .withModules(BaseModule(), ReducedModule())
+            .build().createInjector()
+
+        val config = injector.getInstance(EMConfig::class.java)
+        config.createTests = true
+        config.outputFormat = OutputFormat.PYTHON_UNITTEST
+        config.outputFolder = "$baseTargetFolder/python_requirements"
+        config.outputFilePrefix = "Foo_testPythonRequirements"
+        config.outputFileSuffix = ""
+
+        val solution = getEmptySolution(config)
+
+        //make sure we delete any existing folder from previous test runs
+        val srcFolder = File(config.outputFolder)
+        srcFolder.deleteRecursively()
+
+        val writer = injector.getInstance(TestSuiteWriter::class.java)
+        //write the test suite
+        writer.writeTests(solution, FakeController::class.qualifiedName!!, null)
+
+        // the requirements file should exist
+        val requirementsFile = Paths.get("${config.outputFolder}/requirements.txt")
+        assertTrue(Files.exists(requirementsFile))
+
+        val testContent = String(Files.readAllBytes(requirementsFile))
+        assertTrue(testContent.contains("requests==2.25.1"))
```

Comment:
avoid hard coding constants like `"requests==2.25.1"`. put it as constant variable in the code, and refer to the variable here instead of hard coding

Reply:
Done